src/routes/route-factory.js

```diff
@@ -3,7 +3,7 @@
 const express = require('express');
 const errors = require('@feathersjs/errors');
 
-module.exports = function createRoute(repository) {
+module.exports = function createRoute(repository, createPermissions) {

```

Comment:
`createPermissions` is not a good name, cause implies it's a function but in your case it is not, so I advise to rename it in `permissions`

Also if your parameters numbers rise to more than 2 I recommend to wrap your it in a object.